kura/org.eclipse.kura.web2/src/main/resources/org/eclipse/kura/web/client/messages/Messages.properties

```diff
@@ -771,6 +771,8 @@ wiresDownloadChannels=Download Channels
 wiresUploadChannels=Upload Channels
 wiresNewChannel=New Channel
 wiresDeleteChannel=Delete Channel
+wiresEmptyStringCheck=Import empty values as empty strings.
+wiresEmptyStringCheckHelpText=If checked, empty values contained in the csv will be parsed as empty strings, for all those fields whose type is <String>.
```

Comment:
> If checked, empty values contained in the csv will be parsed as empty strings, for all those fields whose type is <String>

Maybe better like this: "If checked, all <String> typed channel fields that have an empty value in the input csv will be parsed as empty strings."